numpy/core/src/umath/ufunc_object.c

```diff
@@ -1333,7 +1351,8 @@ execute_legacy_ufunc_loop(PyUFuncObject *ufunc,
                     NPY_ORDER order,
                     npy_intp buffersize,
                     PyObject **arr_prep,
-                    ufunc_full_args full_args)
+                    ufunc_full_args full_args,
+                    npy_uint32 *op_flags)
```

Comment:
Docstring for this function should be updated with this new argument

Reply:
Reworked